### ISSUE TITLE
[Admin] Add configuration for display exception menu item

### ISSUE THREAD

**Issue by dannyvw** [Kunstmaan/KunstmaanBundlesCMS#1937]
New feature request to add a configuration option to show the exception menu item under settings. I think we can configure this based on the user role.

**Comment by sandergo90** [comment]
Yes, I agree, we also had some clients that did not want to see such stuff 😄 .
